src/CarteBancaire.svg

```diff
@@ -0,0 +1,17 @@
6OeXOnHTzyFpyOnnlztOOjnkLTkNPkAnrGx5UmVSeWsbGLUmETy1jWKwmbU1isJm1FB6rKTNrGsWqnB6olGNVhRjVRMWspQeqwoxaylB6rKQ9VHGLWFB6onIO1UpB6rDg9VlKQeqw5B6rDg9VlKQeqw5B6rDgtUTg9VhSC1WHB6rDkFqsMeqw5BarDkHqicFqo4LVE5BbrDkFusOD1WHIHVF0kFusKD1WOkgd1hyC3WHBaonILVY6SC3RpwO6ynIwwkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk4Mx7k+UyZjRNmISYhMNmIaXMaNNiNGlxGhT4jRpsRoujnkLSkdHPIWnHTzy52nI6eeQtOOjnlztOR088haUdHPLna6R088haUjo55C05HTzy52nI6eeXO046eeQtOR0c8udpx088hacjo55c7Tjp55C05HTzy52lHRzyFrpI6eeXO0o6MTyBabQtSazamxlNZtY1jVTY8rCjFY2M1FGLWNjNRRjVYTFY2MaqOMarCjFrKUHawmNVFGNUSg9VFGNVhweqwoPVYUjGqw4PVYUHqsOD1UUHqicg9X8YcHqsKC1WHIPVZTkHqsOD1WFBarDg9UacFusOD1WFBarHSQeqynBarDgt1lOQWqJyC1WHBarDg91hyC3WU4HVYcg90XSQOqw5BbrKcFqicgtVjpILdYcgrfaJvEkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk4sx7m+UqXEQ0uY2CXMaNNiNEuI0abEbBpsRo02I0afnllUdPPLnacjp55C0pHRzy52ukdPPIWlI6OeXO046ecC05HRzy52nHTzyFpyOnnlztOOjnkLTkdPPLnTjo55C05HTzy52lHRzyFrpI6eeXO0o6eeQtOOnnlztOR0c8hacdPPLnachRJMa8tSZRM6rGxm1NYY1m1NZrCZtVJi1jWKjjFrK1jVYUYtZSjFrCjGqwoPVRRi1hQeqwoxWU4xqsIeqig7WFB6onB2spwer+ooxaw5B6rKcHawoLVE5B6qOD1ROC1WFB6rDgtVhyD1WHILVYcHqsKQWqx0kHqicFqspSC1WHIPdY6QO6woPVY6SC1ROQO6w5B6rDkFui6SB3WHBarDkFqsdJBbonA6rKcjLGpJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJOXEe5vlEuYhLiENLiNGmzENNiNGmxGjTYjRpsZaLo55C0pHTzy52nHRzyFpyOnnlztOOjnkLTkdXPLnacdHPIWnHTzy505HRzyFpx088udpyOjlkLTjp55c7SkdHLIWnHTzy52nI6eeQtOOjnlztOOnnkLTjo55c7Tjp55c7TkdGZ5ApPWNiqaxU2PKms1jWKmxmsKM2sbGaijFrCjFqKMarCZrGwdqKMaolGLUUYtYQ7WUoxqsKD1UcY1RKD1UUHqsKMarDkHqsOD1WEPVZSg9VhyD1WHB7rKcHqsOQeqw4LVZSkHqsOC1WHB6o05BbrDg9VhQWqw5B6rKcgtVhwWqJyD1WHILVYcFqsOQW6x0kFujTFqsOQWqw5BarHSQOqNKQWqx0kFqsrpBbolINhvEkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkgYj3N8oUuY2CXMaNNiNGlxGjTYjRpsRo02I0a6OeRtUdHPIWnHTzy52nI6OWQtOOnnlztOR08shacdHPLnTkdPLIWlHRzy52ukdPPIWlI6OeXO10jp55C0o6eeXO05HTzyFpyOjnlztOOnnkLTkdHPLnacdPPIWnI6eeXO046OeXO04QWpNZtTXiazqsaxqprLGs1EzaxrFZSYtRM2ooO0Ws6qKMavxlKDrDjGqwoxqpsY1RKD1UcYtYUHqsIeqwoxqspyD1WHB6rCg9VFB6onIPVZTg9VHBaolIPVZTg9Vhweqw4LVYUHqsOD1WOkgtVhQWqw5B6onBbrDgt1lOQeqw5BbrDgtVhwWqJyC1WOkFqsKQW6x0kFqicgt1hwOqw5Baoukgt1hwWqLpGUkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkPMe6Pk+lzGjS4jRpsxo02YoNNiNGlxGjT88qi6eeQtOR0c8hacdPPLnacjp55C046OeXO046eeQtKR0c8udpx088hacjo55c7Tjp55CnI6eWXO046OeQtOR08sudpx0c8hacjp5Zc7Sjp55C10jo55c7SkdPPIWukdPPLnaUhpPIFN6xseVJmprypsYrGs1EzaxsZtYUYtTYzawmLUUY1WUmLWNYtRRjVE4xqooPVYTFZSg9VhRjVVKMaolB6qKDtYcHqsOD1WUoPVYUY1WU4PVYcg9VhwWqylB6rDkHqsOD1RpwWqwpBarDg9VhyC1WGPVYUgtVjpIPVE5BaqOC1ROQeqw4LdYcFqsOQWqw5BbonBarDkFqsdJBarDgdUTkFqsOQWqyukFujTkGwkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkknmI90j5OLmNGmxGjS4jRpsRo02Y0KbEaLo55G0pHRzy52nHTzyFKR08sudrpHRzyFpSOnllztOOnnkLTjo5Zc7Tjp55C046eeXO05HRzyFpx088udpSOjnkLXSOnnlztKR0c8hacdPPLnacjp55C046OeXO05HTzy52nHRiBacbFqTWbU2PE1i1jWdVNjNY1msJi1EzaytjFqKMaqJii2MaqKMarDjFrCjFrCHqooxqiUHqooxawoPVYUHqsOMarKUHqsOD1WHB6qKD1ROQeqynB6rCg9VhyD1WU5B6rDgtVhQWqw5B6rKcHqsOC1RKC1WHIPVZXSQWqw4PdZSkFqsdJBaonBarCkFusdJBarKcFqsOQW6JyC1WOkgt1lOB1ROQeqx0kDusOQVvtE48SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSf/Z"/>
```

Comment:
This is an extremely large blob to be downloading everywhere (and, now, even preloading ahead of page content). There must be a smaller way to encode a similar gradient?

Even better: avoid loading this icon and others like it until necessary.

Reply:
It should only be downloaded individually. This package is set up to isolate imports.

Reply:
This was surfaced by the Chrome coverage tool ([discussed here](https://artsy.slack.com/archives/C05EEBNEF71/p1731018412391149?thread_ts=1731017791.751689&cid=C05EEBNEF71)). If I'm interpreting the results correctly, it seemed to be downloading on the production home page where but was unused.

Reply:
I exported from the [Figma design](https://www.figma.com/design/gZNkyqLT8AU3T61tluVJyB/Artsy-3.1-Design-System?node-id=10968-22354&node-type=frame&t=gf1otpEFIofDjmag-0). Guess it's for the gradient background? Maybe there is a way to optimize?

![Screenshot 2024-11-08 at 5 11 37 PM](https://github.com/user-attachments/assets/6ae298d4-ef16-44ae-ab7d-e6fa3e16847e)

Reply:
Yea, design needs to redraw that to not use an image